NeoML/src/Dnn/Layers/DropoutLayer.cpp

```diff
@@ -50,8 +51,8 @@ void CDropoutLayer::SetDropoutRate( float value )
 	NeoAssert( value >= 0.f && value < 1.f );
 	if( dropoutRate != value ) {
 		dropoutRate = value;
-		if( GetDnn() != 0 ) {
-			destroyDropoutDesc();
+		if( GetDnn() != nullptr) {
+			disableDropoutDesc();
```

Comment:
To disable now cannot help, it should be recreated `onReshape()`, the same in other setters